scss/components/counter.scss

```diff
@@ -11,7 +11,7 @@ $block: #{$fd-namespace}-counter;
   $fd-counter-radius: $fd-border-radius !default;
   $fd-counter-background: fd-color("status", 4) !default;
   $fd-counter-color: fd-color("text", 5) !default;
-  $fd-counter-font-size: 0.785714285714286em !default;
+  $fd-counter-font-size: 0.785714285714286rem !default;
```

Comment:
We need to get rid of the `rem` value as it's possible to make the self-contained and the "all" styles appear in different sizes.  I know we have `rem` values coming from theme designer so that is going to need to be addressed more globally going forward.

Reply:
@greg-a-smith I am not sure how this change had happened. Reverted.